### PR TITLE
Kapacitor RDD

### PR DESCRIPTION
**Pull request by tonerdo**
This PR adds READMEs for the following soon to be built modules:

* `install-kapacitor`: Bash script for installing Kapacitor Enterprise on a node
* `run-kapacitor`: Bash script for configuring and starting Kapacitor Enterprise on a node
* `kapacitor-cluster`: Terraform module for deploying a Kapacitor Enterprise cluster
* `kapacitor-iam-policies`: Terraform module for specifying IAM policies to be attached to each node in a Kapacitor Enterprise cluster
* `kapacitor-security-group-rules`: Terraform module for specifying security group rules to be attached to each node in a Kapacitor Enterprise cluster

### CODE REVIEW

File: examples/tick-ami/tick.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
   },{
     "name": "tick-ami-amazon-linux",
     "ami_name": "{{user `base_ami_name`}}-amazon-linux-example-{{uuid | clean_ami_name}}",
-    "ami_description": "An Amazon Linux 2 AMI that has the TICK installed.",
+    "ami_description": "An Amazon Linux 2 AMI that has the TICK stack installed.",
```
Comment (robmorgan) [inline examples/tick-ami/tick.json]: 👍

File: modules/install-kapacitor/README.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# Kapacitor Install Script
+
+Kapacitor is is a Real-Time Streaming Data Processing Engine
```
Comment (robmorgan) [inline modules/install-kapacitor/README.md]:
```suggestion
Kapacitor is a Real-Time Streaming Data Processing Engine.
```

File: modules/install-kapacitor/README.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+This folder contains a script for installing Kapacitor Enterprise and its dependencies.
+
+This script has been tested on the following operating systems:
+
+* Ubuntu 16.04
```
Comment (robmorgan) [inline modules/install-kapacitor/README.md]:
```suggestion
* Ubuntu 18.04
```
?

File: modules/kapacitor-cluster/README.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+* `ami_id`: Use this parameter to specify the ID of an Kapacitor [Amazon Machine Image 
+  (AMI)](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AMIs.html) to deploy on each server in the cluster. You
+  should install Kapacitor the scripts in the 
+  [install-kapacitor](https://github.com/gruntwork-io/terraform-aws-influx/tree/master/modules/install-kapacitor) module.
+  
+* `user_data`: Use this parameter to specify a [User 
+  Data](http://docs.aws.amazon.com/AWSEC2/latest/UserGuide/user-data.html#user-data-shell-scripts) script that each
+  server will run during boot. This is where you can use the 
+  [run-kapacitor](https://github.com/gruntwork-io/terraform-aws-influx/tree/master/modules/run-kapacitor) 
+  script to configure and run Kapacitor.
```
Comment (robmorgan) [inline modules/kapacitor-cluster/README.md]: Nice 👍

File: modules/kapacitor-cluster/README.md [hunk 5]
```diff
@@ -0,0 +1,127 @@
...
+You can find the other parameters in [variables.tf](variables.tf).
+
+Check out the [examples folder](https://github.com/gruntwork-io/terraform-aws-influx/tree/master/examples) for 
+fully-working sample code.
+
+## Why Kapacitor enterprise?
+
+Unlike the Enterprise edition which distributes the entire Kapacitor deployment across multiple nodes in a cluster,
+the OSS edition is a single binary that can simply be installed on a single instance. This makes a robust module like this one
+quite unnecessary for single instance Kapacitor OSS deployments.
```
Comment (robmorgan) [inline modules/kapacitor-cluster/README.md]: Should we rephrase this paragraph a bit to say Why Kapitor OSS instead?

or 'When should I not use this module?'

I was a bit confused when I first read it as I believe its **for** the Kapacitor Enterprise module.

Comment (tonerdo) [reply modules/kapacitor-cluster/README.md]: Noted!

**Pull request merged**